spec/rubocop/cop/rails/private_transaction_option_spec.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Rails::PrivateTransactionOption, :config do
+  it 'registers an offense when using `joinable: false`' do
+    expect_offense(<<~RUBY)
+      ActiveRecord::Base.transaction(requires_new: true, joinable: false)
+                                                         ^^^^^^^^^^^^^^^ Do not use `ActiveRecord::Base.transaction(joinable: _)`.
+    RUBY
+  end
+
```

Comment:
```suggestion

  it 'registers an offense when using `joinable: false`' do
    expect_offense(<<~RUBY)
      account.transaction(requires_new: true, joinable: false) do
                                              ^^^^^^^^^^^^^^^ Use a negated `requires_new` option instead of the internal `joinable`.
        # ...
      end
    RUBY
  end
```